clients/src/main/resources/common/message/FindCoordinatorRequest.json

```diff
@@ -37,7 +37,7 @@
     { "name": "Key", "type": "string", "versions": "0-3",
       "about": "The coordinator key." },
     { "name": "KeyType", "type": "int8", "versions": "1+", "default": "0", "ignorable": false,
-      "about": "The coordinator key type. (Group, transaction, etc.)" },
+      "about": "The coordinator key type. (Group, transaction, etc.)." },
```

Comment:
I think "(group, transaction, share)" are the three key types supported at the moment.